WordPress/Classes/Networking/ServiceRemoteXMLRPC.m

```diff
@@ -13,12 +13,17 @@ @implementation ServiceRemoteXMLRPC
 
 - (id)initWithApi:(WPXMLRPCClient *)api username:(NSString *)username password:(NSString *)password
 {
+    NSParameterAssert(api != nil);
+    if (_username == nil || _password == nil) {
```

Comment:
I think this was supposed to be `username` and `password`. Right now init will always return `nil`.